next.config.js

```diff
@@ -1,16 +1,40 @@
 /** @type {import('next').NextConfig} */
+require('webpack')
+require('path')
```

Comment:
Why did you add these imports? I don't think they don't do anything because you dont use anything from its namespace.